src/index.js

```diff
@@ -134,6 +136,18 @@ const Spruce = {
                     return comparison[part]
                 }, stores)
             })
+    },
+
+    persistUsing(driver) {
```

Comment:
is there a situation where this could be used to change the persistence method after stores have been initialised, would that cause any weird issues? Worth logging a warning?

Reply:
Yeah, good idea.